db/migrate/20180809101455_merge_duplicate_schools.rb

```diff
@@ -0,0 +1,29 @@
+class MergeDuplicateSchools < ActiveRecord::Migration
+  def up
+    return if School.count == 0 # for Travis
+
+    # This list of schools comes from the spreadsheet at https://jira.nypl.org/browse/MLN-162
```

Comment:
Why are you including a URL to a JIRA ticket in the comments? A ticket can always be edited and changed.